### ISSUE TITLE
Nextflow expecting '=' padding?

### ISSUE THREAD

**Issue by engineer-pat** [seqeralabs/wave-showcase#7]
When running this for GKE after all steps and setup, I'm using the default tutorial.nf from nextflow's site, with the alpine container specified (fulfill demand for containers per process), and get the below. I suspect Nextflow is expecting the '=' base64 padding for the kubernetes secret?

```
WARN: Unable to decode K8s cluster auth token '<TOKEN FROM KUBERNETES SECRET GENERATED VIA gke.yml>'  -- cause: = character not at end of base64 value 

[-        ] process > splitLetters   [  0%] 0 of 1
[-        ] process > convertToUpper -
ERROR ~ Error executing process > 'splitLetters'

Caused by:
[d6/feb18c] process > splitLetters   [100%] 1 of 1, failed: 1
[-        ] process > convertToUpper -
ERROR ~ Error executing process > 'splitLetters'

Caused by:
  Request POST /api/v1/namespaces/<MY_NAMESPACE>/pods returned an error code=403

  {
      "kind": "Status",
      "apiVersion": "v1",      "metadata": {
          
      },
      "status": "Failure",
      "message": "pods is forbidden: User \"system:anonymous\" cannot create resource \"pods\" in API group \"\" in the namespace \"<MY_NAMESPACE>\"",
      "reason": "Forbidden",
      "details": {
          "kind": "pods"
      },
      "code": 403
  }

 -- Check '.nextflow.log' file for details
```

**Comment by pditommaso** [comment]
Any chance to provide the `.nextflow.log` file?

**Comment by engineer-pat** [comment]
Solved: a typo/confusion.

Looked through the .log and found the issue: token tried appeared to have multiple tokens appended to itself, i.e. multiple '='. I initially thought maybe I'd run gke.yml multiple times and for some reason tokens were appended to one file. Actually was caused by the following:

At line 29 in gke.yml (in the chunk below, this is name: wave-sa-token) I'd instead _**wrongly**_ entered the service account name instead of service-account-name-TOKEN. 

Only putting the name seems to override the token that's actually generated and referenced by the service account, permanently, even after fixing the typo; I wasn't able to salvage the account. I deleted the old kubernetes service account, created a new one and applied the corrected gke.yml to the fresh service account. Solves this problem.

---
apiVersion: v1
kind: Secret
metadata:
  namespace: wave-demo
  name: wave-sa-token  ## Be sure this is 'sa-token'
  annotations:
    kubernetes.io/service-account.name: wave-sa 
type: kubernetes.io/service-account-token
...
